### PR TITLE
Added GitEx doc updates from helpdocs

### PR DESCRIPTION
**Pull request by rashminandasahoo**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/platform/10_Git-Experience/configure-git-experience-for-harness-entities.md [hunk 1]
```diff
@@ -77,23 +81,57 @@ Enter a **Name** for your Pipeline.
 Click **Remote**. The additional settings appear to configure Git Experience.
 
 ![](./static/configure-git-experience-for-harness-entities-39.png)
-In **Git Connector**, select or create a Git Connector to the repo for your Project. For steps, see [Code Repo Connectors](/docs/category/code-repo-connectors).
+In **Git Connector**, select or create a Git Connector to the repo for your Project. For steps, see [Code Repo Connectors](../7_Connectors/connect-to-code-repo.md).
```
Comment (SmritiSatya) [inline docs/platform/10_Git-Experience/configure-git-experience-for-harness-entities.md:85]: In **Git Connector**, select (or create) a Git Connector to the repo for your Project.
Would it be relevant if we add the 'or create' within paranthesis?

File: docs/platform/10_Git-Experience/configure-git-experience-for-harness-entities.md [hunk 2]
```diff
@@ -77,23 +81,57 @@ Enter a **Name** for your Pipeline.
...
+![](./static/configure-git-experience-for-harness-entities-41.png) 
+
+For GitHub, the token must have the following scopes: 
+
+![](./static/configure-git-experience-for-harness-entities-42.png)
+
+</details>
+
+Here's an example of a GitHub Connector that has the correct settings:​
+
```
Comment (SmritiSatya) [inline docs/platform/10_Git-Experience/configure-git-experience-for-harness-entities.md:103]: Here's an example of a GitHub Connector with the correct settings:​

File: docs/platform/10_Git-Experience/configure-git-experience-for-harness-entities.md [hunk 3]
```diff
@@ -77,23 +81,57 @@ Enter a **Name** for your Pipeline.
...
-For GitHub, the token must have the following scopes:  
-![](./static/configure-git-experience-for-harness-entities-42.png)Here's an example of a GitHub Connector that has the correct settings:​
 
-![](./static/configure-git-experience-for-harness-entities-43.png)In **Repository**, select your repository. If your repository isn't listed, enter its name since only a select few repositories are filled here.
+<details>
+<summary> IMPORTANT </summary>
+
+Connector must use the Enable API access option and Token
+The Connector must use the Enable API access option and Username and Token authentication. Harness requires the token for API access. Generate the token in your account on the Git provider and add it to Harness as a Secret. Next, use the token in the credentials for the Git Connector.​
+
```
Comment (SmritiSatya) [inline docs/platform/10_Git-Experience/configure-git-experience-for-harness-entities.md:93]: The Connector must use the Enable API access option, Username and Token authentication. Harness requires this token for API access. Generate the token in your account on the Git provider and add it to Harness as a secret.

File: docs/platform/10_Git-Experience/git-experience-overview.md [hunk 4]
```diff
@@ -95,14 +96,22 @@ A Harness Git Connector is used to sync your Harness Project with your Git repo.
 
 You will need a Harness Git Connector to connect with the Git provider and perform operations like generating a webhook. Your Git Connector credentials are used to commit to Git when operations are performed using API.
 
-**Important:** The Connector must use the **Enable API access** option and Username and **Token** authentication. Harness requires the token to access the Git API. Generate the token in your account on the Git provider and add it to Harness as a Secret. Next, use the token in the credentials for the Git Connector.For detailed steps to add a Git Connector, see [Code Repo Connectors](/docs/category/code-repo-connectors).
+
+:::note
+**Important:** The Connector must use the **Enable API access** option and Username and **Token** authentication. Harness requires the token to access the Git API. Generate the token in your account on the Git provider and add it to Harness as a Secret. Next, use the token in the credentials for the Git Connector. For detailed steps to add a Git Connector, see [Code Repo Connectors](../7_Connectors/connect-to-code-repo.md).
```
Comment (SmritiSatya) [inline docs/platform/10_Git-Experience/git-experience-overview.md:101]: **Important:** The Connector must use the **Enable API access** option, Username and **Token** authentication. Harness requires the token to access the Git API. Generate the token in your account on the Git provider and add it to Harness as a secret. Next, use the token in the credentials for the Git Connector. For detailed steps to add a Git Connector, see [Code Repo Connectors](../7_Connectors/connect-to-code-repo.md).

File: docs/platform/10_Git-Experience/import-a-template-from-git.md [hunk 5]
```diff
@@ -41,18 +41,53 @@ You can import a Template from the CI or CD module in Harness.
...
+7. In **Git Connector**, select or create a Git Connector to connect to your Git repo. For steps, see [Code Repo Connectors](https://harness.helpdocs.io/category/xyexvcc206-ref-source-repo-provider).
+   
+   <details>
+   <summary>
+   IMPORTANT: Connector must use the Enable API access option and Token
+
+   </summary>
+   Connector must use the Enable API access option and Token
+   The Connector must use the Enable API access option and Username and Token authentication. Harness requires a token for API access. Generate the token in your account on the Git provider and add it to Harness as a Secret. Next, use the token in the credentials for the Git Connector.​
+
```
Comment (SmritiSatya) [inline docs/platform/10_Git-Experience/import-a-template-from-git.md:63]: https://github.com/harness/developer-hub/pull/340/files#r1068063167 - Likewise

File: docs/platform/10_Git-Experience/configure-git-experience-for-harness-entities.md (hunk 3 above)
Comment (rashminandasahoo) [reply docs/platform/10_Git-Experience/configure-git-experience-for-harness-entities.md:93]: "Username and Token" is a type of authentication.

**Review by SmritiSatya — APPROVED** [review]

**Pull request merged**